src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -508,7 +508,7 @@ private function formatControllerLink($controller, string $anchorText): string
                 $r = new \ReflectionMethod($controller[0], $controller[1]);
             } elseif ($controller instanceof \Closure) {
                 $r = new \ReflectionFunction($controller);
-            } elseif (method_exists($controller, '__invoke')) {
+            } elseif (null !== $controller && method_exists($controller, '__invoke')) {
```

Comment:
I don't think this works well as it will fall back to the `else` clause. I would handle this case as its own clause, probably as the first test in the list.

Reply:
Thanks for your reply. It falls back to the is \is_string case and returns $anchorText but it's also fine for me to handle it in its own condition.